library/system/test/fs_snapshot_test.rb

```diff
@@ -365,4 +401,59 @@ def logger
       end
     end
   end
+
+  describe ".create_snapshot?" do
+    before do
+      Yast.import "Linuxrc"
+    end
+
+    context "when single value is defined on Linuxrc commandline" do
+      it "returns whether given snapshot is requested" do
```

Comment:
Hmm, it seems I was working on the test case before changing the whole method name.